typos

@ecashin

Looks like a couple of typos here, `deiver` on line 20 in the .txt file and `retrivial` on line 10 in the manpage. 

```
$ sed -n '18,22p' /tmp/aoetools/devnodes.txt
If you use an aoe driver from the CORAID website
(http://support.coraid.com/support/linux/) then just use the aoetools that
come bundled with that deiver.  If you don't have udev, make sure you
always load the aoe module with the aoe_dyndevs=0 option for any
driver version above 49.
$ sed -n '8,18p' /tmp/aoetools/aoecfg.8
.SH DESCRIPTION
.IR Aoecfg (8)
sends AoE configuration commands that control the retrivial, conditional
or unconditional setting of AoE configuration strings.  Since configuration
happens before the MAC address of the target is known, the packet is
broadcast.  AoE targets with a matching shelf and slot respond.  Since
the default shelf and slot are the wildcard values 0xffff and 0xff,
with no arguments
.IR aoecfg (8)
will return configuration strings from all targets visible
on the default interface,
$ 
```

Running `make` on the cloned repository resulted in a warning.

```
$ make CFLAGS="-O2 -fPIC -Wl,-s"
+ sed -e 's!@devdir@!/dev/etherd!g' -e 's!@npershelf@!16!g' aoe-discover.in
+ sed -e 's!@devdir@!/dev/etherd!g' -e 's!@npershelf@!16!g' aoe-interfaces.in
+ sed -e 's!@devdir@!/dev/etherd!g' -e 's!@npershelf@!16!g' aoe-mkshelf.in
+ sed -e 's!@devdir@!/dev/etherd!g' -e 's!@npershelf@!16!g' aoe-revalidate.in
+ sed -e 's!@devdir@!/dev/etherd!g' -e 's!@npershelf@!16!g' aoe-flush.in
+ sed -e 's!@devdir@!/dev/etherd!g' -e 's!@npershelf@!16!g' aoe-stat.in
cc -O2 -fPIC -Wl,-s -o aoeping.o -c aoeping.c
cc -O2 -fPIC -Wl,-s -o linux.o -c linux.c
cc -O2 -fPIC -Wl,-s -o aoeping aoeping.o linux.o
cc -O2 -fPIC -Wl,-s -o aoecfg.o -c aoecfg.c
aoecfg.c: In function ‘cfgquery’:
aoecfg.c:133:13: warning: ‘write’ reading 1056 bytes from a region of size 1048 [-Wstringop-overread]
  133 |         if (write(sfd, c, sizeof *c) == -1) {
      |             ^~~~~~~~~~~~~~~~~~~~~~~~
aoecfg.c:111:15: note: source object ‘buf’ of size 1048
  111 |         uchar buf[1024 + sizeof *h];
      |               ^~~
In file included from aoecfg.c:8:
/usr/include/unistd.h:367:16: note: in a call to function ‘write’ declared with attribute ‘access (read_only, 2, 3)’
  367 | extern ssize_t write (int __fd, const void *__buf, size_t __n) __wur
      |                ^~~~~
cc -O2 -fPIC -Wl,-s -o aoecfg aoecfg.o linux.o
cc -O2 -fPIC -Wl,-s -o aoe-sancheck.o -c aoe-sancheck.c
cc -O2 -fPIC -Wl,-s -o aoe-sancheck aoe-sancheck.o -lpthread
$ 
```

--- a/aoecfg.8
+++ b/aoecfg.8
@@ -7,7 +7,7 @@ aoecfg \- manipulate AoE configuration strings
 .fi
 .SH DESCRIPTION
 .IR Aoecfg (8)
-sends AoE configuration commands that control the retrivial, conditional
+sends AoE configuration commands that control the retrieval, conditional
 or unconditional setting of AoE configuration strings.  Since configuration
 happens before the MAC address of the target is known, the packet is
 broadcast.  AoE targets with a matching shelf and slot respond.  Since

--- a/devnodes.txt
+++ b/devnodes.txt
@@ -17,7 +17,7 @@ of the device nodes.)
 
 If you use an aoe driver from the CORAID website
 (http://support.coraid.com/support/linux/) then just use the aoetools that
-come bundled with that deiver.  If you don't have udev, make sure you
+come bundled with that driver.  If you don't have udev, make sure you
 always load the aoe module with the aoe_dyndevs=0 option for any
 driver version above 49.
 

